pom.xml

```
@@ -40,7 +40,7 @@
     </developers>
 
     <properties>
-        <maven.compiler.target>7</maven.compiler.target>
+        <maven.compiler.target>9</maven.compiler.target>
```

The agent should remain Java 7 compatible
```suggestion
        <maven.compiler.target>7</maven.compiler.target>
```